iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -99,37 +87,68 @@ class new_ConversationViewController: FullScreenViewController, CollectionViewIn
             self.conversationController.deleteMessage(message.id)
         }
     }
-}
-
-extension new_ConversationViewController: UICollectionViewDelegate {
 
+    // MARK: - UICollectionViewDelegate
 
+    /// If true, the conversation controller is currently loading messages.
+    @Atomic private var isLoadingMessages = false
```

Comment:
atomic?

Reply:
@Benjamindodgson Yeah this is to prevent any possible threading issues. Only one thread can access the var at a time.